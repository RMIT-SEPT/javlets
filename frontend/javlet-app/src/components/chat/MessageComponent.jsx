import React, { Component } from 'react'

class MessageComponent extends Component{
    state = {  }
render(){
   return(
    <div className="chat">
        <h4>{this.props.sender}</h4>
      <p> <img className='w3-circle' alt="" src={'https://loremflickr.com/100/100/person' + this.props.message} />{this.props.message}</p>
    </div>
   );
}
}
export default MessageComponent;