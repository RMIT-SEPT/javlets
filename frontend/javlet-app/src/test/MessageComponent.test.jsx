import React from 'react';
import { shallow } from 'enzyme';
import MessageComponent from '../components/chat/MessageComponent.jsx';

  it('MessageComponent SNAPSHOTt', () => {
    const component = shallow(<MessageComponent message="Test message" datetime="28/09/2019 14:57:20" sender="User" />);
  
    expect(component).toMatchSnapshot();
  });