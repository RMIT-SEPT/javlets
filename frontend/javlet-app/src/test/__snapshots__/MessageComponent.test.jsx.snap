// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageComponent SNAPSHOTt 1`] = `
<div
  className="chat container darker"
>
  <img
    alt=""
    className="right"
    src="https://img.icons8.com/cotton/64/000000/name--v2.png"
  />
  <span
    className="name"
  >
    User
  </span>
  <p
    className="w3-blue w3-round-large"
  >
    Test message
     
  </p>
  <span
    className="w3-tiny"
  >
    28/09/2019 14:57:20
  </span>
</div>
`;
